### ISSUE TITLE
"Kernel died, restarting..." in jupyter qtconsole on Windows 7

### ISSUE THREAD

**Issue by kokostek** [jupyter/qtconsole#319]
### Steps to reproduce

- Download python 3.7.1 distribution for windows from official site. (https://www.python.org/ftp/python/3.7.1/python-3.7.1-amd64.exe)
- Install at c:\Python37 for all users.
- Create virtual environment at `c:\Python\envs\test-env`:
```
C:\>cd c:\Python37
C:\Python37>python -m venv c:\Python\envs\test-env
```
- Activate virtualenv and install `jupyter` with `pip`:
```
C:\Python37>cd c:\
C:\>Python\envs\test-env\Scripts\activate.bat
(test-env) C:\>pip install jupyter pyqt5
```
- Try to launch `qtconsole`:
```
(test-env) C:\>jupyter qtconsole
```
- Get `Kernel died, restarting` message repeating over and over with 80% probability.

### Remark about 80% probability
Some times `qtconsole` launches with no problem and I can see the IPython prompt. But most times it fails.

### Versions and environment
Jupyter QtConsole 4.4.3
```
(test-env) C:\>python -c "import IPython; print(IPython.sys_info())"
{'commit_hash': '523ed2fe5',
 'commit_source': 'installation',
 'default_encoding': 'utf-8',
 'ipython_path': 'c:\\Python\\envs\\test-env\\lib\\site-packages\\IPython',
 'ipython_version': '7.2.0',
 'os_name': 'nt',
 'platform': 'Windows-7-6.1.7601-SP1',
 'sys_executable': 'c:\\Python\\envs\\test-env\\Scripts\\python.exe',
 'sys_platform': 'win32',
 'sys_version': '3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:57:15) [MSC v.1915 64 bit (AMD64)]'}
```

**Comment by kokostek** [comment]
I am really confused with absence of logs or any additional info on this. Somehow I have managed to get this message in qtconsole window:
'''
c:\Python\envs\qtconsole\lib\site-packages\ipykernel\parentpoller.py:116: UserWarning: Parent poll failed.  If the frontend dies,
                the kernel may be left running.  Please let us know
                about your system (bitness, Python, etc.) at
                ipython-dev@scipy.org
  ipython-dev@scipy.org""")
'''
It showed up after I typed `exit` in successfully launched qtconsole. But it was one time thing and I couldn't repeat it afterwards.

**Comment by kokostek** [comment]
After some playing around with versions I came up with this working configuration:
```
pip install qtconsole==4.3.1 ipykernel==4.8.2 ipython==6.4.0 pyqt5==5.10.1
```
Version of `pyqt5` is choosen becuse of this [issue](https://github.com/jupyter/qtconsole/issues/291).
Versions of other packages are taken form another virtualenv on my PC, which I used before.

Also, qtconsole 4.5.0.dev0 seems to work fine.

It must be something wrong with 4.4.3.
